### PR TITLE
Add Docker compose Xray example

### PR DESCRIPTION
**Pull request by crazy-max**
See #115

### CODE REVIEW

File: docker-compose/xray/xray.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+version: "2"
```
Comment (eldada) [inline docker-compose/xray/xray.yml]: Use version 3

File: docker-compose/xray/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Xray Docker Compose Examples
+
+This directory provides some examples that show different ways to run Xrazy with Docker Compose.
```
Comment (eldada) [inline docker-compose/xray/README.md]: `Xrazy` -> `Xray`

File: docker-compose/xray/prepareHostEnv.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This file is for preparing all the needed files and directories on the host.
+
+source ./.env
```
Comment (eldada) [inline docker-compose/xray/prepareHostEnv.sh]: Check file `.env` exists

**Comment by eldada** [comment]
Also need to undo unchanged files that just have permissions changes on them.

**Comment by crazy-max** [comment]
:tada: :tada: :tada:

**Pull request merged**